### PR TITLE
A more detailed intro in the README.

### PR DESCRIPTION
**Pull request by andersio**
It adds a table of "core reactive primitives" with a very brief description plus a metaphor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,36 @@
...
-it’s easy to declaratively chain and combine them together, with less spaghetti
+it’s easy to declaratively compose them together, with less spaghetti
 code and state to bridge the gap.
 
-For more information about the concepts in ReactiveSwift, see the [Framework
-Overview][].
+### Core Reactive Primitives
+| Primitive | Description |
+| --------- | ------------|
+| `Signal` | `Signal` is a unilateral stream of events.<br /><br />It is like a live TV feed - you can observe and react to the content, but you cannot have a side effect on the live feed. |
```
Comment (mdiep) [inline README.md]: I don't think _unilateral_ is the right word here. _Unidirectional_, maybe?

The ` - ` should should be replaced by `—` here and below.

File: README.md [hunk 2]
```diff
@@ -9,21 +9,108 @@
...
+__ReactiveSwift__ offers composable, declarative and flexible primitives that are built around the grand concept of ___streams of values over time___.
 
- * Delegate methods
- * Callback blocks
- * Notifications
- * Control actions and responder chain events
- * [Futures and promises](https://en.wikipedia.org/wiki/Futures_and_promises)
- * [Key-value observing](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/KeyValueObserving/KeyValueObserving.html) (KVO)
+These primitives can be used to uniformly represent common Cocoa and generic programming patterns that are fundamentally an act of observation, e.g. delegate pattern, callback closures, notifications, control actions, responder chain events, [futures, promises](https://en.wikipedia.org/wiki/Futures_and_promises) and [key-value observing](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/KeyValueObserving/KeyValueObserving.html) (KVO).
```
Comment (sharplet) [inline README.md]: I think "futures/promises" would read better than "futures, promises"

File: README.md [hunk 3]
```diff
@@ -9,21 +9,108 @@
...
+channel.observeValues { programme in ... }
+```
+
+#### `Event`: the basic transfer unit of an event stream.
+A `Signal` may have any arbitrary number of events carrying a value, following by an eventual terminal event of a specific reason.
+
+It is like a frame in a one-time live feed — seas of data frames carry the visual and audio data, but the feed would eventually be terminated with a special frame to indicate "end of stream".
+
+#### `SignalProducer`: deferred work that creates a stream of value.
+`SignalProducer` defers work — of which the output is represented as a stream of value — until it is started. For every invocation to start the `SignalProducer`, a new `Signal` is created and the deferred work is subsequently invoked.
```
Comment (mdiep) [inline README.md]: > stream of value **_s_**

File: README.md [hunk 4]
```diff
@@ -9,21 +9,108 @@
...
-Overview][].
+### Core Reactive Primitives
+#### `Signal`: a unidirectional stream of events.
+The owner of a `Signal` has unilateral control of the event stream. Observers may register their interests in the future events at any time, but the observation would have no side effect on the stream or its owner.
+
+It is like a live TV feed — you can observe and react to the content, but you cannot have a side effect on the live feed or the TV station.
+
+```swift
+let channel: Signal<Programme, NoError> = tvStation.channelOne
+channel.observeValues { programme in ... }
```
Comment (mdiep) [inline README.md]: RAC uses US English, so this should be _program_.

File: README.md [hunk 5]
```diff
@@ -24,16 +24,16 @@ The owner of a `Signal` has unilateral control of the event stream. Observers ma
...
 ```
 
 #### `Event`: the basic transfer unit of an event stream.
 A `Signal` may have any arbitrary number of events carrying a value, following by an eventual terminal event of a specific reason.
 
 It is like a frame in a one-time live feed — seas of data frames carry the visual and audio data, but the feed would eventually be terminated with a special frame to indicate "end of stream".
 
-#### `SignalProducer`: deferred work that creates a stream of value.
+#### `SignalProducer`: deferred work that creates a stream of values.
 `SignalProducer` defers work — of which the output is represented as a stream of value — until it is started. For every invocation to start the `SignalProducer`, a new `Signal` is created and the deferred work is subsequently invoked.
```
Comment (mdiep) [inline README.md]: _stream of value_ is repeated in the body text too. 🙈

**Pull request merged**